doc-content/enterprise-only/release-notes/rn-7.12.1-fixed-issues-ref.adoc

```diff
@@ -0,0 +1,62 @@
+[id='rn-7.12.1-fixed-issues-ref']
+= Fixed issues in {PRODUCT} 7.12.1
+
+{PRODUCT} 7.12.1 provides increased stability and fixed issues listed in this section.
+
+== {CENTRAL}
+
+* When you have two data objects, form generation enters into an infinite loop when it reaches to the first nested form [https://issues.redhat.com/browse/RHPAM-3366[RHPAM-3366]]
+* An inaccurate alert about duplicate rule name after copying a rule in {CENTRAL} [https://issues.redhat.com/browse/RHPAM-2871[RHPAM-2871]]
+* In test scenarios designer, `today()` and `now()` functions are not evaluated correctly [https://issues.redhat.com/browse/RHDM-1816[RHDM-1816]]
+* When you delete a column from a guided decision table and try to validate that guided decision table, you receive an error [https://issues.redhat.com/browse/RHDM-1813[RHDM-1813]]
+* When you are converting a guided decision table to XLS, backslash is added before the double quotes and you receive an error [https://issues.redhat.com/browse/RHDM-1723[RHDM-1723]]
+
+== Build and assembly
+
+* In Spring Framework, versions 5.3.0 - 5.3.10, 5.2.0 - 5.2.17, and an older unsupported versions, it is possible for a user to provide malicious input to cause the insertion of additional log entries [https://issues.redhat.com/browse/RHPAM-4098[RHPAM-4098]]
+
+ifdef::PAM[]
+
+== {PROCESS_ENGINE_CAP}
+
+* Boundary timer is not triggered after a rollback [https://issues.redhat.com/browse/RHPAM-4102[RHPAM-4102]]
+* Incorrect response for REST service when `org.kie.server.bypass.auth.user` system property is used with `JAASUserGroupCallbackImpl` implementation [https://issues.redhat.com/browse/RHPAM-4087[RHPAM-4087]]
+* When you use `ClusteredJobFailOverListener` method, you receive system errors caused by `ConcurrentModificationException` exception [https://issues.redhat.com/browse/RHPAM-4069[RHPAM-4069]]
+* Certificate based authentication is required between {KIE_SERVER} and process instance migration tool [https://issues.redhat.com/browse/RHPAM-4038[RHPAM-4038]]
+* Smart router is sending requests to a stopped {KIE_SERVER} [https://issues.redhat.com/browse/RHPAM-4035[RHPAM-4035]]
+* The `ProcessServiceImpl.getAvailableSignals` must obtain process instance in read-only mode to avoid race condition [https://issues.redhat.com/browse/RHPAM-3967[RHPAM-3967]]
```

Comment:
```suggestion
* `ProcessServiceImpl.getAvailableSignals` must obtain process instances in read-only mode to avoid race condition [https://issues.redhat.com/browse/RHPAM-3967[RHPAM-3967]]
```